### PR TITLE
Malaria profile

### PR DESCRIPTION
**Pull request by manutarus**
Edit malaria confirmation
Add upcoming service on malaria profile

### CODE REVIEW

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- opensrp-chw/src/main/java/org/smartregister/chw/interactor/MalariaUpcomingServiceInteractor.java  3
- opensrp-chw/src/main/java/org/smartregister/chw/interactor/ChildUpcomingServicesInteractor.java  1
- opensrp-chw/src/main/java/org/smartregister/chw/util/Utils.java  1
- opensrp-chw/src/main/java/org/smartregister/chw/interactor/MalariaProfileInteractor.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426625&bid=17796037)

File: opensrp-chw/src/main/java/org/smartregister/chw/activity/FamilyProfileActivity.java [hunk 1]
```diff
@@ -13,13 +13,17 @@
 import org.smartregister.chw.core.activity.CoreFamilyProfileMenuActivity;
 import org.smartregister.chw.core.activity.CoreFamilyRemoveMemberActivity;
 import org.smartregister.chw.core.utils.CoreConstants;
+import org.smartregister.chw.dao.MalariaDao;
 import org.smartregister.chw.fragment.FamilyProfileActivityFragment;
 import org.smartregister.chw.fragment.FamilyProfileDueFragment;
 import org.smartregister.chw.fragment.FamilyProfileMemberFragment;
+import org.smartregister.chw.malaria.activity.BaseMalariaProfileActivity;
```
Comment (keymane) [inline opensrp-chw/src/main/java/org/smartregister/chw/activity/FamilyProfileActivity.java:20]: Codacy Issue found: [Avoid unused imports such as 'org.smartregister.chw.malaria.activity.BaseMalariaProfileActivity'](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426625)

File: opensrp-chw/src/main/java/org/smartregister/chw/activity/FamilyProfileActivity.java [hunk 2]
```diff
@@ -13,13 +13,17 @@
 import org.smartregister.chw.core.activity.CoreFamilyProfileMenuActivity;
 import org.smartregister.chw.core.activity.CoreFamilyRemoveMemberActivity;
 import org.smartregister.chw.core.utils.CoreConstants;
+import org.smartregister.chw.dao.MalariaDao;
```
Comment (keymane) [inline opensrp-chw/src/main/java/org/smartregister/chw/activity/FamilyProfileActivity.java:16]: Codacy Issue found: [Avoid unused imports such as 'org.smartregister.chw.dao.MalariaDao'](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426625)

**Comment by manutarus** [comment]
out of date

**Pull request closed without merging**